README.md

```diff
@@ -1,47 +1,70 @@
-# A minimal Astro site with Sanity Studio
+# Minimal Astro site with Sanity Studio
 
 This starter uses [Astro](https://astro.build/) for the front end and [Sanity](https://sanity.io/) to handle its content.
 
-## Featuring
+![Screenshot of the SvelteKit and Sanity starter template](https://placehold.co/1680x1050)
 
-- How to fetch content as data from [the Sanity Content Lake](https://www.sanity.io/docs/datastore)
-- How to render block content with [Portable Text](https://www.sanity.io/docs/presenting-block-text)
-- A [Sanity Studio](https://www.sanity.io/docs/sanity-studio) to create and edit content
-- How to crop and render images with [Sanity Image URLs](https://www.sanity.io/docs/presenting-images)
+## Features
 
-## Prerequisites
+- Fetch content as data from [the Sanity Content Lake](https://www.sanity.io/docs/datastore)
+- Render block content with [Portable Text](https://www.sanity.io/docs/presenting-block-text)
+- Manage and create content with the intuitive [Sanity Studio](https://www.sanity.io/docs/sanity-studio).
+- Crop and render images with [Sanity Image URLs](https://www.sanity.io/docs/presenting-images)
 
-- [Node.js](https://nodejs.org/en/) (v16.12 or later)
+## Demo
 
-## Getting started
+[Try the live demo](https://template-astro-clean.sanity.build/)
 
-Run the following commands
+## Getting Started
 
-1. `npm install` to install dependencies
-2. `npx sanity@latest init --env`, this will:
+### Prerequisites
 
-   - ask you to select or create a Sanity project and dataset
-   - output a `.env` file with appropriate variables
-   - _(or use `sanity init --env` if you have the CLI installed)_
+- [Node.js](https://nodejs.org/en/) - Astro requires v18.17.1 or v20.3.0, v22.0.0 or higher. ( v19 and v21 are not supported.)
 
-3. `npm run dev` to start the development server
+### Setup Steps
 
-Your Astro app should now be running on [http://localhost:4321/](http://localhost:4321/) and Studio on [http://localhost:4321/admin](http://localhost:4321/admin).
+#### 1. Install the template
 
-### Add content
+Run the command in your Terminal to initialize this template on your local computer.
+
+See the documentation if you are [having issues with the CLI](https://www.sanity.io/help/cli-errors).
+
+```shell
+npm create sanity@latest --template sanity-template-astro-clean
```

Comment:
@RostiMelk Is this right?

Reply:
Missing org prefix here and npm create flag. Should be:
`npm create sanity@latest -- --template sanity-io/sanity-template-astro-clean`